### PR TITLE
[AAP-25503] Refactor instance tests

### PR DESCRIPTION
**Pull request by nixocio**
Update instance tests and instance groups.

### CODE REVIEW

File: cypress/e2e/awx/infrastructure/instance-groups/instanceGroupsK8s.cy.ts [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+    cy.contains('nav[aria-label="Breadcrumb"]', 'Instances').should('exist');
+    cy.contains('nav[aria-label="Breadcrumb"]', instance.hostname).should('exist');
+    cy.contains('nav[aria-label="Breadcrumb"]', 'Details').should('exist');
+    cy.intercept('POST', awxAPI`/instances/${instance.id.toString()}/health_check/`).as(
+      'runHealthCheck'
+    );
+    cy.getByDataCy('run-health-check').click();
+    cy.wait('@runHealthCheck')
+      .then((response) => {
+        expect(response.response?.statusCode).to.be.true;
```
Comment (vidyanambiar) [inline cypress/e2e/awx/infrastructure/instance-groups/instanceGroupsK8s.cy.ts:259]:
```suggestion
        expect(response.response?.statusCode).to.eql(200);
```

I got a failure while testing this locally against https://awx.ci-ui-20240827.gcp.testing.ansible.com
`'can visit the details page of an instance nested inside an instance group and run health check on it'`
image

Comment (nixocio) [reply cypress/e2e/awx/infrastructure/instance-groups/instanceGroupsK8s.cy.ts]: @vidyanambiar fixed.

**Comment by nixocio** [comment]
> There was also this failure locally in instances.cy.ts: `can run a health check on an Instance in the instance details page and assert the expected results` image

I could not see that one. Trying again.


image

**Review by vidyanambiar — APPROVED** [review]
As discussed the flaky test will be separately addressed.

**Pull request merged**